### PR TITLE
Generate IDs in the actions, and then use those to update the state later

### PR DESCRIPTION
**Pull request by isaacseymour**
@jackfranklin

### CODE REVIEW

File: app/reducers.js [hunk 1]
```diff
@@ -39,46 +37,38 @@ import uuid from 'uuid';
 
 const initialState = {
   destinations: Map({
-    [uuid.v1()]: Map({ postcode: 'WC1X 9QZ' }),
+    specialId: Map({ postcode: 'WC1X 9QZ' }),
```
Comment (jackfranklin) [inline app/reducers.js]: why this change?

**Comment by jackfranklin** [comment]
:+1:

File: app/reducers.js (hunk 1 above)
Comment (isaacseymour) [inline app/reducers.js]: Means we don't need to `import uuid` here, and doesn't seem worth doing so for this one use ¯\_(ツ)_/¯
